databricks/resource_databricks_workspace_conf_test.go

```diff
@@ -0,0 +1,44 @@
+package databricks
+
+// Preview feature: https://docs.databricks.com/security/network/ip-access-list.html
+// REST API: https://docs.databricks.com/dev-tools/api/latest/ip-access-list.html#operation/create-list
+
+import (
+	"testing"
+)
+
+func TestWorkspaceConfCreateOrUpdate(t *testing.T) {
+	// d, err := ResourceTester(t, []HTTPFixture{
```

Comment:
no large commented chunks of code in this repository, please :)

Reply:
heh, np, I was explicitly asking for help, there.  It was difficult to mix the ResourceTester with the Option/Builder pattern.

Reply:
Rewrote